content/develop/concepts/configuration/theming.md

```diff
@@ -21,10 +21,10 @@ in the `[theme]` section of a `.streamlit/config.toml` file.
 
 ```toml
 [theme]
-primaryColor="#F63366"
+primaryColor="#FF4B4B"
 backgroundColor="#FFFFFF"
 secondaryBackgroundColor="#F0F2F6"
```

Comment:
@mayagbarnes I'm trying to verify the default colors here per a GitHub issue in the docs. I'm struggling to confirm `secondaryBackgoundColor`. Digging through the code, I am expecting `secondaryBg` to be in `themeColor.ts` but I'm not seeing it. Do you know what this is currently? (I'm asking since you were the last to edit themeColors.ts and I'm hoping you know. 😅)

Reply:
Light theme `secondaryBg` = "#f0f2f6" 
( ^ For light theme its inherited from emotionBaseTheme/themeColors.ts)
Dark theme `secondaryBg` = "#262730"

Hope thats what you were looking for!